metrotranscom/doorway
# feat: update county labels to show full name in listings list

# Pull Request Template

## Description

- Display the full and translated county name for listing cards

I tried also updating the labels for the landing and listing page search for the counties but it's not a straightforward change. The label is currently both used for display and for functionality (to see when a county is selected) so using the translations breaks the search functionality. The county search will need a rewrite and I would rather not tackle that during the bug bash.

## How Can This Be Tested/Reviewed?

- Go to the listings page on the public site
- Verify that the county name includes the word "County"
- Switch to another language and verify that the county name is translated

County names in English 
![image](https://github.com/metrotranscom/doorway/assets/67125998/e22a5643-361c-4a04-b447-23531ff1b9ed)

County names in Spanish
![image](https://github.com/metrotranscom/doorway/assets/67125998/ea0f8425-a79b-48d5-897d-0cea0ed5bbb0)
